hw/ip/prim/dv/prim_lfsr/data/prim_lfsr_cov_excl.el

```diff
@@ -0,0 +1,49 @@
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[24].i_prim_lfsr
+ANNOTATION: "[EXTERNAL/UNR] external seed and entropy inputs are disabled in DV, these assertions are fully covered in FPV"
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+ANNOTATION: "[EXTERNAL/UNR] external seed and entropy inputs are disabled in DV, these assertions are fully covered in FPV"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094 1520894529"
+INSTANCE: prim_lfsr_tb.gen_duts[24].i_prim_lfsr
+ANNOTATION: "EXTERNAL: This point is fully covered in the FPV bench for prim_lfsr, so it is ok to exclude from DV bench."
+Condition 8 "1510648413" "(lfsr_en_i && (gen_max_len_sva.cnt_q == gen_max_len_sva.cmp_val)) 1 -1" (1 "01")
+ANNOTATION: "EXTERNAL: This point is fully covered in the FPV bench for prim_lfsr, so it is ok to exclude from DV bench."
+Condition 6 "3622507969" "(lfsr_en_i && lockup) 1 -1" (1 "01")
+ANNOTATION: "EXTERNAL: This point is fully covered in the FPV bench for prim_lfsr, so it is ok to exclude from DV bench."
+Condition 6 "3622507969" "(lfsr_en_i && lockup) 1 -1" (3 "11")
+ANNOTATION: "EXTERNAL: This point is fully covered in the FPV bench for prim_lfsr, so it is ok to exclude from DV bench."
+Condition 5 "4214161107" "((lfsr_en_i && lockup) ? '0 : ((lfsr_en_i && (gen_max_len_sva.cnt_q == gen_max_len_sva.cmp_val)) ? '0 : (lfsr_en_i ? ((gen_max_len_sva.cnt_q + 1'b1)) : gen_max_len_sva.cnt_q))) 1 -1" (2 "1")
+ANNOTATION: "EXTERNAL: This point is fully covered in the FPV bench for prim_lfsr, so it is ok to exclude from DV bench."
+Condition 3 "1026054222" "(lfsr_en_i && lockup) 1 -1" (1 "01")
+ANNOTATION: "EXTERNAL: This point is fully covered in the FPV bench for prim_lfsr, so it is ok to exclude from DV bench."
+Condition 3 "1026054222" "(lfsr_en_i && lockup) 1 -1" (3 "11")
+ANNOTATION: "EXTERNAL: This point is fully covered in the FPV bench for prim_lfsr, so it is ok to exclude from DV bench."
+Condition 2 "2655198170" "((lfsr_en_i && lockup) ? DefaultSeed : (lfsr_en_i ? next_lfsr_state : lfsr_q)) 1 -1" (2 "1")
+CHECKSUM: "1437046094 2214500533"
+INSTANCE: prim_lfsr_tb.gen_duts[24].i_prim_lfsr
+ANNOTATION: "EXTERNAL: This point is fully covered in the FPV bench for prim_lfsr, so it is ok to exclude from DV bench."
+Branch 1 "864650299" "(lfsr_en_i && lockup)" (0) "(lfsr_en_i && lockup) 1,-,-"
+ANNOTATION: "EXTERNAL: This point is fully covered in the FPV bench for prim_lfsr, so it is ok to exclude from DV bench."
+Branch 0 "780369097" "seed_en_i" (1) "seed_en_i 0,1,-"
+ANNOTATION: "EXTERNAL: This point is fully covered in the FPV bench for prim_lfsr, so it is ok to exclude from DV bench."
+Branch 0 "780369097" "seed_en_i" (0) "seed_en_i 1,-,-"
+CHECKSUM: "1437046094 528004685"
+INSTANCE: prim_lfsr_tb.gen_duts[24].i_prim_lfsr
+ANNOTATION: "EXTERNAL: This point is fully covered in the FPV bench for prim_lfsr, so it is ok to exclude from DV bench."
+Block 33 "1545234335" "state = DefaultSeed;"
```

Comment:
Would you please update these to look uniformly like this:
```
[CATEGORY-TAG] <additional explanation if required>
```
The point of tagging is to provide an easy way to grep for the categories of exclusions.